example/gardener-local/gardenlet/values-ha.yaml

```diff
@@ -0,0 +1,102 @@
+global:
+  gardenlet:
+    config:
+      gardenClientConnection:
+        kubeconfigSecret:
+          name: gardenlet-kubeconfig
+          namespace: garden
+        bootstrapKubeconfig:
+          name: gardenlet-kubeconfig-bootstrap
+          namespace: garden
+          kubeconfig: |
+            apiVersion: v1
+            kind: Config
+            current-context: gardenlet-bootstrap
+            clusters:
+            - cluster:
+                insecure-skip-tls-verify: true
+                server: https://gardener-local-control-plane:6443
+              name: default
+            contexts:
+            - context:
+                cluster: default
+                user: gardenlet-bootstrap
+              name: gardenlet-bootstrap
+            users:
+            - name: gardenlet-bootstrap
+              user:
+                token: 07401b.f395accd246ae52d
+      controllers:
+        shoot:
+          reconcileInMaintenanceOnly: true
+      featureGates:
+        HVPA: true
+        HVPAForShootedSeed: true
+        ManagedIstio: true
+        APIServerSNI: true
+        ReversedVPN: true
+        ShootCARotation: true
+        ShootSARotation: true
+        CopyEtcdBackupsDuringControlPlaneMigration: true
+        HAControlPlanes: true
+        DefaultSeccompProfile: true
+        CoreDNSQueryRewriting: true
+      logging:
+        enabled: true
+        loki:
+          enabled: true
+        shootNodeLogging:
+          shootPurposes:
+          - "infrastructure"
+          - "production"
+          - "development"
+          - "evaluation"
+      seedConfig:
+        apiVersion: core.gardener.cloud/v1beta1
+        kind: Seed
+        metadata:
+          name: local
+          labels:
+            seed.gardener.cloud/multi-zonal: "true"
```

Comment:
Should we name this `local-ha` to make it clear and not confuse it with the other `local` seed?

Also, do I see it correctly that everything in this file is similar to `values.yaml` except for this label? If yes, can we rather only add this special configuration instead of duplicating the entire file (similar to https://github.com/gardener/gardener/blob/master/example/gardener-local/gardenlet/values-kind2.yaml)?